packages/intro/types.ts

```diff
@@ -30,7 +31,7 @@ export interface IntroProps {
   /**
    * Text following the heading element.
    */
-  description: string
+  description: string | React.ReactNode
```

Comment:
Could we add this to the props.js file as well for the swingset props tabler?
https://github.com/hashicorp/react-components/blob/main/packages/intro/props.js#L33

Reply:
good catch, updated https://github.com/hashicorp/react-components/pull/814/commits/6097387bb238f162caaa168e036b5b97bc3b672a